src/python/requirements-app.txt

```diff
@@ -1,2 +1,4 @@
 ### your custom requirements here
-#protobuf==3.13.0
\ No newline at end of file
+graphistry[ai] == 0.29.1
```

Comment:
Broadly... we shouldn't have any app requirements? Maybe we should add a comment in the file to clarify these are for users to add deps